test/unit/FieldsetDate.test.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-env mocha */
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import FieldsetDate from '../../src/app/components/Filters/FieldsetDate';
+import Actions from './../../src/app/actions/Actions.js';
+
+describe('FieldsetDate', () => {
+  describe('Default', () => {
+    const selectedFacets = {
+      language: ['lang:fr', 'lang:fr'],
+    };
+    let component;
+
+    before(() => {
+      component = mount(<FieldsetDate selectedFacets={selectedFacets} />);
+    });
+
+    after(() => {
+      component.unmount();
+    });
+
+    it('should render a fieldset.', () => {
+      expect(component.find('fieldset').length).to.equal(1);
+    });
+
+    it('should render a legend with text Date.', () => {
+      expect(component.find('legend').length).to.equal(1);
+      expect(component.find('legend').text()).to.equal('Date');
+    });
+
+    it('should have an input and label for each list item.', () => {
```

Comment:
`list item` - There's no `ul` or `li` in the component.

Reply:
Yeah I missed this one.